common/taskana-common/src/main/java/pro/taskana/common/internal/test/InitOracleSchema.java

```diff
@@ -0,0 +1,70 @@
+package pro.taskana.common.internal.test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+import pro.taskana.common.api.exceptions.SystemException;
+
+/**
+ * Helper Class to create a proper ORACLE SCHEMA
+ *
+ * <p>This class is only used in TEST classes. DON'T USE IT IN PRODUCTION CODE.
```

Comment:
Can we move this somehow into test code or would this mean we have to duplicate the code?

Reply:
Duplicate into test-api and common-test

Reply:
done